File: rafael/arrays/SequenciaDecrescente.cs
class SequenciaDecrescente 
{
	static void Main() 
	{
		int[] array = new int[10];

		for (int i = 0; i < array.Length; i++) 
		{
			array[i] = array.Length - 1 - i;
		}

		for (int i = 0; i < array.Length; i++) 
		{
			System.Console.WriteLine(array[i]);
		}
	}
}

File: rafael/arrays/SequenciaDecrescente.java
class SequenciaDecrescente {
	public static void main(String[] args) {
		int[] array = new int[10];

		for (int i = 0; i < array.length; i++) {
			array[i] = array.length - 1 - i;
		}

		for (int i = 0; i < array.length; i++) {
			System.out.println(array[i]);
		}
	}
}
